src/vendor_upstream/dom/DOMMouseMoveTracker.js

```diff
@@ -72,14 +76,36 @@ class DOMMouseMoveTracker {
         'mouseout',
         this.onMouseEnd
       );
+
+      if (this._isTouchEnabled && !this._eventTouchStartToken &&
```

Comment:
Should this be outside the if statement above?

Reply:
Yes, good catch.